index.html

```diff
@@ -56,17 +57,110 @@ <h2>Multi-Post Stories</h2>
+                    <ul>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="project" type="button">See Project </button>
+                </div>
+                <div class="card-2">
+                    <h2>Professional Art <br>Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="project" type="button">See Project </button>
+                </div>
+                <div class="card-2">
+                    <h2>Professional Art <br>Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="project" type="button">See Project </button>
+                </div>
+                <div class="card-2">
+                    <h2>Professional Art <br>Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="project" type="button">See Project </button>
+                </div>
+                <div class="card-2">
+                    <h2>Professional Art <br>Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="project" type="button">See Project </button>
+                </div>
+                <div class="card-2">
+                    <h2>Professional Art <br>Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="project" type="button">See Project </button>
                 </div>
             </div> 
+        </section>
+    </div>
+    <div class="container-3">
```

Comment:
- [ ] As suggested by the previous code reviewer, I kindly suggest that you change this`div` tag to a`section` tag for [semantic](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) purposes.